.github/dependabot.yml

```diff
@@ -20,3 +20,8 @@ updates:
     schedule:
       interval: weekly
     open-pull-requests-limit: 100
+  - package-ecosystem: yarn
```

Comment:
Not sure this is right actually, check https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem